src/stac_utils/truthy.py

```diff
@@ -2,5 +2,12 @@
 
 
 def truthy(thing: str, true_values: list = None) -> bool:
+    """
+    Evaluates whether a string is `True`
```

Comment:
All of the following will evaluate as true: 1, Yes, Y, Yeah, T, True, true
Most other things will evaluate as false, including None and undefined.